### PR TITLE
updated Italian after translator is done, added a new-synonyms for it…

### PR DESCRIPTION
**Pull request by leokim-l**
…alian as well

Let me know what you think :)

### CODE REVIEW

File: babelon/hp-it.new-synonyms.tsv
Comment (matentzn) [inline babelon/hp-it.new-synonyms.tsv:1]: You have to overwrite https://github.com/obophenotype/hpo-translations/blob/main/babelon/hp-it.synonyms.tsv for this to work, and maintain the first two rows as they are..

File: babelon/hp-it.babelon.tsv [hunk 1]
```diff
@@ -1,2900 +1,3489 @@
 source_language	source_value	subject_id	predicate_id	translation_language	translation_value	translation_status	translator	translator_expertise	comment	translation_date
-en	Hydrocephalus	HP:0000238	rdfs:label	it	Idrocefalo	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Severe global developmental delay	HP:0011344	rdfs:label	it	Grave ritardo globale dello sviluppo	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Retinal hemorrhage	HP:0000573	rdfs:label	it	Emorragia retinica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Intracranial hemorrhage	HP:0002170	rdfs:label	it	Emorragia intracranica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Delayed speech and language development	HP:0000750	rdfs:label	it	Ritardo nello sviluppo del linguaggio e della parola	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Bulbous nose	HP:0000414	rdfs:label	it	Naso bulboso	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
-en	Hypotonia	HP:0001252	rdfs:label	it	Ipotonia	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Multicystic kidney dysplasia	HP:0000003	rdfs:label	it	Displasia renale multicistica	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
```
Comment (matentzn) [inline babelon/hp-it.babelon.tsv:2]: What does "preview" mean in the comment?

File: babelon/hp-it.babelon.tsv [hunk 2]
```diff
@@ -1,2900 +1,3489 @@
...
+en	Recurrent urinary tract infections	HP:0000010	rdfs:label	it	Infezioni ricorrenti del tratto urinario	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Neurogenic bladder	HP:0000011	rdfs:label	it	Vescica neurogena	CANDIDATE	DeepL	ALGORITHM	preview	2024-05-08
+en	Urinary urgency	HP:0000012	rdfs:label	it	Urgenza urinaria	OFFICIAL	0000-0003-4174-0526	PHYSICIAN		2024-11-26
```
Comment (matentzn) [inline babelon/hp-it.babelon.tsv]: Can we do `orcid:0000-0003-4174-0526` instead of `0000-0003-4174-0526` for sustainability?

**Review by matentzn — CHANGES_REQUESTED** [review]

File: babelon/hp-it.babelon.tsv (hunk 1 above)
Comment (leokim-l) [reply babelon/hp-it.babelon.tsv:2]: I don't know, this is not a new line, it comes from the previously existing file...  see
https://github.com/obophenotype/hpo-translations/blob/c14c23fc4e4acba7b47db15b447d919faef03a10/babelon/hp-it.babelon.tsv#L1010

File: babelon/hp-it.babelon.tsv (hunk 2 above)
Comment (leokim-l) [reply babelon/hp-it.babelon.tsv]: done in https://github.com/obophenotype/hpo-translations/pull/20/commits/4d224123c493799b6a0140656d04d9741b6853ad

File: babelon/hp-it.new-synonyms.tsv
Comment (leokim-l) [reply babelon/hp-it.new-synonyms.tsv:1]: done in https://github.com/obophenotype/hpo-translations/pull/20/commits/4d224123c493799b6a0140656d04d9741b6853ad

File: babelon/hp-it.babelon.tsv (hunk 1 above)
Comment (matentzn) [reply babelon/hp-it.babelon.tsv:2]: Peter must have added it. Ok, not important, I don't think! leave it as is now!

File: babelon/hp-it.new-synonyms.tsv
Comment (matentzn) [inline babelon/hp-it.new-synonyms.tsv:1]: Remove file from PR?

**Review by matentzn — CHANGES_REQUESTED** [review]

**Review by matentzn — APPROVED** [review]

**Pull request merged**